tests/packages/test_api.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2019 Red Hat, Inc
+# see the LICENSE file for license
+#
+
+import pytest
+
+
+def test_delete_released_package(
+        client, valid_manifests_archive, endpoint_packages,
+        mocked_packages_delete_quay_io):
+    """Test REST API for deleting released operators manifest packages"""
+
+    rv = client.delete(
+        endpoint_packages.url_path,
+    )
+
+    assert rv.status_code == 200, rv.get_json()
+    expected = {
+        'organization': endpoint_packages.org,
+        'repo': endpoint_packages.repo,
+        'deleted': ["0.0.1"],
+    }
+    assert rv.get_json() == expected
+
+
+@pytest.mark.parametrize('endpoint', [
+    '/packages/organization-X/repo-Y',
+    '/packages/organization-X/repo-Y/1.0.1',
+])
+@pytest.mark.parametrize('method', [
+    'GET', 'PATCH' 'PUT', 'HEAD', 'POST', 'TRACE',
+])
+def test_method_not_allowed(client, endpoint, method):
+    """Specified endpoints currently support only DELETE method, test if other
+    HTTP methods returns proper error code
+
+    Method OPTIONS is excluded from testing due its special meaning
+    """
+    rv = client.open(endpoint, method=method)
+    assert rv.status_code == 405
```

Comment:
```suggestion
    assert rv.status_code == requests.codes.method_not_allowed
```

In general code readability could be improved by using [names from `requests.codes`](http://docs.python-requests.org/en/master/api/#status-code-lookup) instead of hardcoded values.

Reply:
ok

Reply:
updated